package.json

```diff
@@ -171,6 +173,7 @@
         "randomstring": "^1.1.5",
         "react": "^16.13.1",
         "react-animate-height": "^2.0.17",
+        "react-awesome-query-builder": "4",
```

Comment:
Version 5 has come out in the meantime and an update right away might be worthwhile if it's not too much pain.
If only because they have renamed `Like` to `Contains` in the dropdown, which I like better :)